src/features/search/helpers/useSortedSearchCategories/useSortedSearchCategories.ts

```diff
@@ -16,22 +18,27 @@ export function categoriesSortPredicate(a: MappingOutput, b: MappingOutput): num
   return positionA - positionB
 }
 
-export const useSortedSearchCategories = (
-  onPressCategory: OnPressCategory
-): ListCategoryButtonProps => {
+export const useSortedSearchCategories = (): ListCategoryButtonProps => {
   const searchGroupLabelMapping = useSearchGroupLabelMapping()
   const categories = useAvailableCategories()
+  const { data } = useSubcategories()
+  const hasAThematicSearch = useHasAThematicPageList()
 
+  const navigateTo = (facetFilter: SearchGroupNameEnumv2) => {
+    const searchTabConfig = getNavigateToConfig(
+      hasAThematicSearch.includes(facetFilter) ? 'ThematicSearch' : 'SearchResults',
+      {
+        offerCategories: [facetFilter],
+        isFullyDigitalOffersCategory: (data && isOnlyOnline(data, facetFilter)) || undefined,
```

Comment:
```suggestion
        isFullyDigitalOffersCategory: data && isOnlyOnline(data, facetFilter),
```
Je pense que ça suffit, c'est un booléen et s'il est `undefined` il sera considéré comme `false`.
PI il n'est utilisé que là : https://github.com/pass-culture/pass-culture-app-native/blob/7514f1b5f2624fed45e3d4ea94d57c23bc91c2c6/src/libs/algolia/fetchAlgolia/buildAlgoliaParameters/buildOfferSearchParameters.ts#L53